Feature/class03 (#52)
devopsacademyau/academy: classes/03class/exercises/c03-docker07/README.md

# ECS (c03-docker07)

> TBD

Reviewer: WIP

Author: waiting for the lb/asg class to check how far I can go with this exercise.

# ECS (c03-docker07)

> Fir this exercise we're going to play with ECS and will create a Cluster, a Service and a Task definition and we'll access the container through the internet.

Reviewer: ```suggestion
For this exercise we're going to play with ECS and create a Cluster, a Service, and a Task definition. Then we'll access the container through the internet.
```

> This exercise will cost some money, so if you don;t want to spend any money here, don;t do it.

Reviewer: ```suggestion
This exercise will cost some money, so it is OPTIONAL. If you don't want to spend any money, don't do it.
```
